python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_restraint_search.py

```diff
@@ -326,53 +343,66 @@ def _analyse(
         method="MDRestraintsGenerator",
         append_to_ligand_selection="",
         receptor_selection_str="protein and name CA C N",
-        cutoff=10 * _angstrom,  # In Angstrom
+        force_constant=None,
+        cutoff=8 * _angstrom,
```

Comment:
Why change this?

Reply:
I think 8 A seemed enough in most cases and cut the search time slightly from 10 A. However, I'll revert to 10 A for consistency.

Reply:
Yes, I think this would be good. Thanks.